### ISSUE TITLE
TODO tag not found in python code in Jupyter notebook? 

### ISSUE THREAD

**Issue by hopezh** [Gruntfuggly/todo-tree#548]
It seems the TODO tag in the comment of a python code in Jupyter notebook is not found by TODO Tree in vscode, as shown below, whereas the same tag in the comment of a javascrpt is correctly found by TODO Tree. 

May I ask if this is a known issue? 

![Screen Shot 2021-09-04 at 23 16 59 PM](https://user-images.githubusercontent.com/23435810/132099494-fe60bfce-c492-4f63-a0c1-961ba6cee106.png)

The settings related to **TODO Tree** in my `settings.json` file: 
```

  "todo-tree.tree.showScanModeButton": true,
  "todo-tree.tree.scanMode": "open files",

  "todo-tree.highlights.defaultHighlight": {
    "gutterIcon": true
  },

  "todo-tree.highlights.customHighlight": {
    "TODO": {
      "icon": "check",
      "foreground": "black",
      "background": "white",
      "iconColour": "white"
    },

    "NOTE": {
      "icon": "note",
      "foreground": "white",
      "background": "gray",
      "iconColour": "gray"
    },

    "COMMENT": {
      "icon": "comment",
      "foreground": "white",
      "background": "gray",
      "iconColour": "gray"
    },

    "FIXME": {
      "foreground": "black",
      "background": "yellow",
      "iconColour": "yellow"
    },

    "BUG": {
      "foreground": "black",
      "background": "red",
      "iconColour": "red"
    },

    "[+]": {
      "icon": "list-unordered",
      "foreground": "black",
      "background": "00ff00",
      "opacity": 20,
      "iconColour": "yellow"
      // "type": "whole-line"
    },

    "[-]": {
      "icon": "arrow-right",
      "foreground": "black",
      "background": "ffa500",
      "opacity": 20,
      "iconColour": "orange"
    },

    "[.]": {
      "icon": "chevron-right",
      "foreground": "black",
      "background": "0000ff",
      "opacity": 20,
      "iconColour": "blue"
    },

    "[x]": {
      "icon": "check",
      "foreground": "white",
      "background": "green",
      "iconColour": "green"
    },

    "CLOSED": {
      "icon": "eye-closed"
    }
  },

  "todo-tree.general.tags": [
    "BUG",
    "HACK",
    "FIXME",
    "TODO",
    "NOTE",
    "COMMENT",
    "[+]",
    "[-]",
    "[.]",
    "[x]",
    "CLOSED"
  ],
```

**Comment by Gruntfuggly** [comment]
Are you using the default setting for`todo-tree.regex.regex` ? Also could you post the plain text code of the notebook?

**Comment by hopezh** [comment]
@Gruntfuggly, thanks for your prompt reply. 

As shown above I don't have anything related to `todo-tree.regex.regex` specified in my `settings.json` file. (Do I need to specify anything related to regex to solve this issue?) 

The plain text content of the jupyter notebook is show below:

```
{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "source": [
    "# TODO tag in python comment in jupyter notebook "
   ],
   "outputs": [],
   "metadata": {}
  }
 ],
 "metadata": {
  "orig_nbformat": 4,
  "language_info": {
   "name": "python",
   "version": "3.9.5"
  },
  "kernelspec": {
   "name": "python3",
   "display_name": "Python 3.9.5 64-bit ('udacity-deep-learning': conda)"
  },
  "interpreter": {
   "hash": "8c213064a5e6d0c923d7a177f02e2c9bf407a06ccdb685cc176520478ec1133f"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
```

... which is rendered as the following in vscode: 
![Screen Shot 2021-09-05 at 12 25 13 PM](https://user-images.githubusercontent.com/23435810/132115314-3fc9cd4d-aa18-4dc8-893a-3006de3e13a8.png)

**Comment by Gruntfuggly** [comment]
Hi - I've just tried this out with your file and it seems to work as expected for me?

![image](https://user-images.githubusercontent.com/376658/132409425-518ed953-fd86-45a2-a515-15a635e44011.png)
![image](https://user-images.githubusercontent.com/376658/132409481-847015c7-6c0e-4a61-8024-1cb674e2c0f4.png)

**Comment by Gruntfuggly** [comment]
However - I've just noticed your scan mode is set to "Open Files". If you change it to "Workspace and Open Files" I think it will find it. 

It probably doesn't find it as a "Open File" because the notebook is not rendered as a normal file. I will have a look to see if there is a way to find it.

**Comment by Gruntfuggly** [comment]
If you add `vscode-notebook-cell` to `todo-tree.general.schemes` it will find it in "Open Files" scan mode too.

**Comment by hopezh** [comment]
Thanks, @Gruntfuggly. 

Yes, this fixed the issue. 

![Image 3](https://user-images.githubusercontent.com/23435810/132442337-b80eef3c-88ed-43a8-b25d-fa3901d4bc01.png)

![Image 4](https://user-images.githubusercontent.com/23435810/132442341-893d4c44-8fca-4bc2-a52c-d95a02ee271b.png)

**Comment by ImahnShekhzadeh** [comment]
Hi @hopezh and @Gruntfuggly, sorry for reopening the issue, but I noticed I can only view the TODO's, FIXME's, etc. when I am in the *right* cell where I defined the tags in the first place. However, I would like to see the TODO's, FIXME's, etc. of *all* cells (i.e. of the entire Jupyter NB) at once, is that possible? 

The settings I added to my `settings.json` file are 
```jason
"todo-tree.tree.scanMode": "open files", 
    "todo-tree.general.schemes": [
        "file",
        "ssh",
        "untitled",
        "vscode-notebook-cell"
    ],
```
but it did not solve the issue...
